### PR TITLE
Better test prefixes

### PR DESCRIPTION
**Pull request by ariporad**
OK @sindresorhus, This is half of #373, and half of the fixes #370. I think the git history is pretty clean, but let me know if there's anything I should fix.

Before:
Screenshot
After:
Screenshot
Files:
Screenshot

### CODE REVIEW

**Comment by vdemedes** [comment]
@ariporad Could you describe what this PR improves/fixes and provide an example of before/after output?

**Comment by ariporad** [comment]
@vdemedes: on mobile atm, but I'll post a screenshot when I get back. Basically, it properly displayes subdirectories with the default reporter.

File: api.js [hunk 1]
```diff
@@ -195,7 +192,7 @@ function handlePaths(files) {
 		files = [
 			'test.js',
 			'test-*.js',
-			'test/*.js'
+			'test'
```
Comment (sindresorhus) [inline api.js]: This should be in the other PR.

**Comment by ariporad** [comment]
@vdemedes: I uploaded some screenshots.

File: api.js (hunk 1 above)
Comment (ariporad) [inline api.js]: I'll fix that.

Comment (ariporad) [inline api.js]: @sindresorhus: Fixed.

**Comment by vdemedes** [comment]
Great, thanks!

@sindresorhus @jamestalmage

**Comment by sindresorhus** [comment]
LGTM

**Comment by ariporad** [comment]
@vdemedes: Everything else look good about this PR?

**Comment by ariporad** [comment]
@sindresorhus: Sorry if this is stupid, but why [isn't there a `LGTM` status check][Screenshot] for this PR?

[Screenshot]: https://www.dropbox.com/s/cox3pupyehwy331/Screenshot%202015-12-27%2018.24.52.png?dl=1 "Screenshot: There isn't a LGTM status check"

**Comment by sindresorhus** [comment]
@ariporad → https://github.com/sindresorhus/ava/commit/437e88f203791f9828b0d746e4c36e192079197f

**Comment by ariporad** [comment]
@sindresorhus: Ok, thanks.

**Comment by ariporad** [comment]
@sindresorhus: I'm going to merge this, is that OK?

**Pull request merged**